code/game/objects/structures/railings.dm

```diff
@@ -12,6 +12,9 @@
 	var/currently_climbed = FALSE
 	var/mover_dir = null
 
+/obj/structure/railing/get_climb_text()
+	return "<span class='info'>You can <b>Click-Drag</b> someone to [src] to put them over [src.p_them()] after a short delay.</span>"
```

Comment:
You can't actually click drag people onto railings or tables, only yourself. I think we should probably change these two texts